docs/usage/getting-started/private-packages.md

```diff
@@ -265,6 +265,18 @@ Renovate doesn't support reading `npmRegistries` and `npmScopes` from `.yarnrc.y
 Renovate updates `npmRegistries` in `.yarnrc.yml` with resolved `hostRules` before running Yarn.
 For Renovate to overwrite existing `npmRegistries` entry, the key should match the `matchHost` without the protocol (`http:` or `https:`) and with the trailing slash.
```

Comment:
```suggestion
For Renovate to overwrite existing `npmRegistries` entry, the key should match the `matchHost` minus the protocol (`http:` or `https:`) plus the trailing slash.
```
Would this be more clear?

Reply:
An example would be ideal

Reply:
The example below is for this.

Reply:
Can you also add the hostrule to the sample.

I thought this sentence is about the hostrule and not the `.yarnrc.yaml` file. 🤔

Reply:
It refers to the example directly above this, but to be more clear 😄